packages/data-provider/src/services/submittedDataService.ts

```diff
@@ -0,0 +1,41 @@
+import { Dependencies } from '../config/config.js';
+import submittedRepository from '../repository/submittedRepository.js';
+import submittedUtils from '../utils/submittedDataUtils.js';
+import { paginationOps } from '../utils/types.js';
+
+const service = (dependencies: Dependencies) => {
+	const LOG_MODULE = 'SUBMITTED_DATA_SERVICE';
+	const submittedDataRepo = submittedRepository(dependencies);
+	const { logger } = dependencies;
+	return {
+		getSubmittedDataByCategory: async (categoryId: number, paginationOps: paginationOps) => {
+			const { getSubmittedDataByCategoryIdPaginated } = submittedDataRepo;
+			const { parseSubmittedData } = submittedUtils(dependencies);
+
+			const data = await getSubmittedDataByCategoryIdPaginated(categoryId, paginationOps);
+
+			logger.debug(LOG_MODULE, `Retrieved '${data?.length}' Submitted data on categoryId '${categoryId}'`);
+
+			if (!data || data.length === 0) return;
+
+			return parseSubmittedData(data);
+		},
+		getSubmittedDataByOrganization: async (categoryId: number, organization: string, paginationOps: paginationOps) => {
+			const { getSubmittedDataByOrganizationPaginated } = submittedDataRepo;
+			const { parseSubmittedData } = submittedUtils(dependencies);
+
+			const data = await getSubmittedDataByOrganizationPaginated(categoryId, organization, paginationOps);
```

Comment:
This function throws an error but that error is not being handled. This is fine when it is invoked from an express router that is protected by an error handler, but if we ever invoke this function from another source this can result in a server crash.

Is this how we want to handle it?